docs/architecture/adr-009-v6-ics27-msgserver.md

```diff
@@ -0,0 +1,110 @@
+# ADR 009: ICS27 message server addition
```

Comment:
```suggestion
# ADR 009: ICS27 Message Server Addition
```

Reply:
I believe we follow google style recommendations which only capitalizes the first word in a title cc @crodriguezvega should I capitalize Message?